src/components/Navbar/Logo.tsx

```diff
@@ -1,20 +1,25 @@
+import { ThemeOption } from 'constants/theme/theme-option'
 import { ThemeContext } from 'contexts/themeContext'
+import Image from 'next/image'
 import { useContext } from 'react'
-
-import { ThemeOption } from 'constants/theme/theme-option'
 import { twMerge } from 'tailwind-merge'
 
 export default function Logo({ className }: { className?: string }) {
   const { forThemeOption } = useContext(ThemeContext)
 
+  if (!forThemeOption) return null
+
   return (
-    <img
-      className={twMerge('h-10', className)}
-      src={forThemeOption?.({
-        [ThemeOption.light]: '/assets/juice_logo-ol.png',
-        [ThemeOption.dark]: '/assets/juice_logo-od.png',
-      })}
-      alt="Juicebox logo"
-    />
+    <div className={twMerge('relative h-9 w-7 md:h-10 md:w-8', className)}>
+      <Image
```

Comment:
**suggestion** i have some vague memory that it's better to specify the exact width and height to `next/image` (for perf reasons). true?

Reply:
Oh is it? It's really hard to know as theyve changed next/image so much recently. I think `fill` is fine. If you don't specific fill, you must set width/height or it doesn't work at all.

I'm kinda anti next/image because its super unintuitive tbh, but this is such a small simple change